packages/frontend/editor-ui/src/utils/experiments.ts

```diff
@@ -55,3 +60,67 @@ export const trackTemplatesClick = (source: TemplateClickSource) => {
 		source,
 	});
 };
+
+/*
+ * Personalized templates
+ */
+
+export const isPersonalizedTemplatesExperimentEnabled = () => {
```

Comment:
Have you test this on cloud env? Maybe I am wrong, but using store like this outside of Vue context, may read wrong value for `userIsTrialing` (e.g. access before the store is initialized and not reacting to changes afterwards).
I suggest moving this method to `templates.store`